plugins/status/metrics.go

```diff
@@ -0,0 +1,55 @@
+package status
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	pluginStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
```

Comment:
I'm not an expert with prometheus, so please bear with me on the naive question: Is it common to use gauges like that? From the docs,

> Gauge is a Metric that represents a single numerical value that can arbitrarily go up and down.
>
> A Gauge is typically used for measured values like temperatures or current memory usage, but also "counts" that can go up and down, like the number of running goroutines.

Now, I'm not sure how "status" and the unix nano metrics below fit into this. 🤔

Reply:
It is common, as the Gauge is an int64 and can be set as any value it is perfect for timestamp.
For example:
https://github.com/kubernetes/kube-state-metrics/blob/master/docs/cronjob-metrics.md the metric kube_cronjob_next_schedule_time
https://github.com/prometheus/node_exporter the metric node_time_seconds

Reply:
Hey @srenatus , I haven’t made this clear what I was willing to do with each metric, my bad sorry. 
The table below has how I use each metric (The queries are just examples).
| Metric name | Metric type | Description | What questions it answers (dashboard) | alert
| --- | --- | --- |--- |--- |
| plugin_status_gauge | gauge | Number of plugins by name and status. |- How many plugins I have?  'sum by(instance)(plugin_status_gauge{})'. - which instance has the plugin *? 'plugin_status_gauge{name="x"}' |-Has any plugin unhealthy? 'plugin_status_gauge{status!~"OK,NOT_READY"} > 0' |
| bundle_loaded_counter | counter | Number of bundles loaded with success. |How Many successful  pooling I Do Per minute? 'sum by(name)(rate(bundle_loaded_counter{}[1m]))'||
| bundle_failed_load_counter | counter | Number of bundles that failed to load. |How Many unsuccessfull  pooling pooling I Do Per minute ?'sum by(instance, name, code, message)(rate(bundle_failed_load_counter{}[1m]))'| The Erro-rate is > 10% ? '((sum by(instance, name)(rate(bundle_failed_load_counter{}[1m]))+sum by(instance, name)(rate(bundle_loaded_counter{}[1m])))/sum by(instance, name)(rate(bundle_failed_load_counter{}[1m]))>10'|
| last_bundle_request | gauge | Last bundle request in UNIX nanoseconds. |How old was the last bundle request '(time()-((max by(name)(last_bundle_request{instance=~"$instance", name=~"$bundle"})/1e+9)))'||
| last_success_bundle_activation | gauge |Last successfully bundle activation in UNIX nanoseconds. |How old was the last successfully bundle activation '(time()-((max by(name)(last_success_bundle_activation{instance=~"$instance", name=~"$bundle"})/1e+9)))'||
| last_success_bundle_download | gauge|Last bundle request in UNIX nanoseconds. |How old was the last successfully bundle download '(time()-((max by(name)(last_success_bundle_download{instance=~"$instance", name=~"$bundle"})/1e+9)))'||
| last_success_bundle_request | gauge | Last successfully bundle request in UNIX nanoseconds. |How old was the last successfully bundle request '(time()-((max by(name)(last_success_bundle_request{instance=~"$instance", name=~"$bundle"})/1e+9)))'||
| bundle_loading_duration_ns | histogram | A histogram of duration for bundle loading. |What is my 95% bundle loading duration? 'histogram_quantile(0.95, sum(rate(bundle_loading_duration_ns_bucket{instance="$instance",name=~"$bundle"}[1m])) by (le,name))'||

Thinking about which questions The status metrics should answer, what questions would you do?

Reply:
For exemple part of my dash : ![image](https://user-images.githubusercontent.com/1127365/151217382-2110d950-f2d4-471e-9c6e-a5b1de6259ed.png)